uiflow/src/main/java/id/co/veritrans/sdk/uiflow/activities/BBMMoneyActivity.java

```diff
@@ -221,7 +221,7 @@ private void performTransaction() {
         SdkUIFlowUtil.showProgressDialog(BBMMoneyActivity.this, false);
 
         //Execute transaction
-        veritransSDK.paymentUsingBBMMoney();
+//        veritransSDK.paymentUsingBBMMoney();
```

Comment:
Just remove this line or the entire class. BBM Money is completely deprecated I think.